src/util/coordinate_calculation.cpp

```diff
@@ -22,6 +22,11 @@ namespace coordinate_calculation
 
 namespace
 {
+
+// earth radius varies between 6,356.750-6,378.135 km (3,949.901-3,963.189mi)
+// The IUGG value for the equatorial radius is 6378.137 km (3963.19 miles)
+const constexpr long double EARTH_RADIUS = 6372797.560856;
```

Comment:
I don't think this has to be long double? Why do we need it?

Reply:
You are right up to 15 digits should be fine.